build/dvd.sh

```diff
@@ -47,14 +47,18 @@ setup_mtree ${STAGEDIR}/work
 setup_entropy ${STAGEDIR}/work
 
 UEFIBOOT=
+LEGACYBOOT=
 
 if [ -n "${PRODUCT_UEFI}" -a -z "${PRODUCT_UEFI%%*"${SELF}"*}" ]; then
-	UEFIBOOT="-o bootimage=i386;${STAGEDIR}/efiboot.img"
+	UEFIBOOT="-o bootimage=efi;${STAGEDIR}/efiboot.img"
```

Comment:
To comment on this again: I don't want to test this on amd64 for regression when I'm supposed to be building the next opnsense release and people wonder why it may not work as before.

Reply:
Understandable, amd64 now has the sysid i386 also for the efi bootimage.